### PR TITLE
Echo yes to plink command in Run-LinuxCmd to avoid prompts

### PR DESCRIPTION
**Pull request by glewis9999**

### CODE REVIEW

**Comment by juhlee-microsoft** [comment]
@glewis9999 please add more context in Description. We did not see prompt issue so far. This is infra change.

**Comment by vyadavmsft** [comment]
Gordon reported in some scenario IP changes in every reboot and prompts for "yes" and this needs to be echoed.

**Comment by juhlee-microsoft** [comment]
@vyadavmsft I suggest this change will be validated against the most of distro images. This change is in infra not test case. We have not seen the issue before.

**Comment by LiliDeng** [comment]
In function GetAndCheck-KernelLogs, it will upload a random file to the VM to avoid enter yes when ssh a new fresh IP, see https://github.com/LIS/LISAv2/blob/master/Libraries/TestLogs.psm1#L264

**Comment by chvalean** [comment]
> @glewis9999 please add more context in Description. We did not see prompt issue so far. This is infra change.

@glewis9999 I would suggest to provide more context and some logs that will show the need for this.
Otherwise this seems unlikely and the issue you're facing might be somewhere else, but IMO the changes in this PR are not required.

File: Libraries/TestHelpers.psm1 [hunk 1]
```diff
@@ -368,7 +368,8 @@ Function Run-LinuxCmd([string] $username,[string] $password,[string] $ip,[string
 			$logCommand = "`"$MaskedCommand`""
 		}
 	}
-	Write-LogInfo ".\Tools\plink.exe -t -pw $password -P $port $username@$ip $logCommand"
+	$output = Write-Output "yes" | & ".\Tools\plink.exe" -t -pw $password -P $port $username@$ip $logCommand
```
Comment (iamshital) [inline Libraries/TestHelpers.psm1:371]: Even if you want to do echo yes, this is not the correct place.
This is just doing Write-LogInfo. The plink.exe is not getting invoked here.

**Comment by iamshital** [comment]
> Gordon reported in some scenario IP changes in every reboot and prompts for "yes" and this needs to be echoed.

@glewis9999 
If after every reboot, the IP is changing, the best way to handle this is - Upload a random file to machine after the reboot. 
Upload uses pscp.exe with echo 'yes', and this will add the remote machine key to trusted one. 
So, later, you don't need to do echo 'yes' in RunLinuxCmd.

**Comment by LiliDeng** [comment]
Usually reboot won't make public IP change, shutdown=>start will change public IP on azure.

**Comment by glewis9999** [comment]
Thank you for the comments.  I think I can close out this pull request and use the suggestions in this thread.

**Pull request closed without merging**